packages/core/src/amazonqFeatureDev/controllers/chat/messenger/messenger.ts

```diff
@@ -46,7 +46,20 @@ export class Messenger {
         )
     }
 
-    public sendErrorMessage(errorMessage: string, tabID: string, retries: number, phase?: SessionStatePhase) {
+    public sendErrorMessage(
+        errorMessage: string,
+        tabID: string,
+        retries: number,
+        phase?: SessionStatePhase,
+        monthlyLimitError: boolean = false
+    ) {
+        if (monthlyLimitError) {
+            this.dispatcher.sendErrorMessage(
+                new ErrorMessage(`Sorry, we encountered a problem when processing your request.`, errorMessage, tabID)
```

Comment:
I feel like we can make this error a bit more descriptive, at this point we know that they've hit a monthly limit error so we can probably notify them that they can't continue

Reply:
Updated error message based on inputs from UX.